iree/compiler/Translation/CodegenPasses/HLOToLinalgOnBuffers.cpp

```diff
@@ -84,6 +84,30 @@ static ArrayAttr getParallelAndReductionIterAttrs(Builder b, unsigned nLoops,
   return b.getArrayAttr(attrs);
 }
 
+/// Returns a zero value for the given type. Returns null value if the given
+/// type is not supported.
+Value getZeroForType(Type type, Location loc, OpBuilder &builder) {
```

Comment:
builder.getZeroAttr will do this for you

Reply:
Actually I added that API sometime ago. Now I cannot recall it. LOL